--- heap-shot/HeapShot.Gui.Widgets/gtk-gui/HeapShot.Gui.Widgets.ReferenceTreeViewer.cs.orig	2010-05-07 20:06:58.000000000 +0200
+++ heap-shot/HeapShot.Gui.Widgets/gtk-gui/HeapShot.Gui.Widgets.ReferenceTreeViewer.cs	2010-05-07 20:17:20.905783337 +0200
@@ -69,7 +69,7 @@
             this.entryFilter.CanFocus = true;
             this.entryFilter.Name = "entryFilter";
             this.entryFilter.IsEditable = true;
-            this.entryFilter.InvisibleChar = '●';
+            this.entryFilter.InvisibleChar = '\u25cf';
             this.boxFilter.Add(this.entryFilter);
             Gtk.Box.BoxChild w3 = ((Gtk.Box.BoxChild)(this.boxFilter[this.entryFilter]));
             w3.Position = 0;
